src/main/java/net/starlark/java/eval/Starlark.java

```diff
@@ -591,11 +591,12 @@ public static String trimDocString(String docString) {
       return "";
     }
     // First line is special: we fully strip it and ignore it for leading spaces calculation
-    String firstLineTrimmed = StringModule.INSTANCE.strip(lines.get(0), NONE);
+    String firstLineTrimmed =
+        StringModule.INSTANCE.stripSemantics(lines.get(0), NONE, StarlarkSemantics.DEFAULT);
```

Comment:
Relying on `StarlarkSemantics.DEFAULT` is often a code smell. In this case, it means that flipping the flag in any way that doesn't also change its hardcoded default value will create an inconsistency between how U+A0 is handled in docstring content vs how the .bzl gets decoded.

The proper fix would require taking a StarlarkSemantics as an argument to this method. That looks like a nuisance, so it's fine to add a TODO.